blazor/datagrid/columns.md

```diff
@@ -511,6 +511,69 @@ In the following code sample, you can prevent default filter query generation us
 
 > You can find the fully working sample [here](https://github.com/SyncfusionExamples/blazor-datagrid-prevent-query-generation-for-foriegnkey-column).
 
+### Customize filter UI in foreignkey column
+
+A custom filtering UI can be rendered using the [FilterTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_FilterTemplate) feature of the Grid. The following example demonstrates the way to create a custom filtering UI in the foreign column.
```

Comment:
same as this PR. https://github.com/syncfusion-content/blazor-docs/pull/1195/files. remove one PR. keep one alone. but there is no foreign key relationships between tables.

Reply:
Need to modify the entire content. "Custom Component can be rendered in the Filter UI of Foreign Key column using the FilterTemplate feature of Grid. 

The following example demonstrates the way to create a custom filtering UI by rendering SfDropDownList component in the foreign key column"